.env.example

```diff
@@ -0,0 +1,11 @@
+PORT=
+MONGO_URI=
+REDISUSER=
+REDISPASSWORD=
+REDISHOST=
+REDISPORT=
```

Comment:
We paused use or Redis, maybe we will get back to it in the future, when we find a good use case for it. We forgot to scrap out code related to Redis, but it's fine.

Reply:
U could told me to make changes 
but now pr is merged :(

Reply:
That is not a big issue. At some point, those env variables will be needed

Reply:
Ah yeah got that,
yup that also make sense